packages/react-charting/src/utilities/utilities.ts

```diff
@@ -961,9 +961,8 @@ export function domainRangeOfVSBCNumeric(
 ): IDomainNRange {
   const xMin = d3Min(points, (point: IDataPoint) => point.x as number)!;
   const xMax = d3Max(points, (point: IDataPoint) => point.x as number)!;
-  // barWidth / 2 - for to get tick middle of the bar
-  const rMax = margins.left! + barWidth / 2;
-  const rMin = width - margins.right! - barWidth / 2;
+  const rMax = margins.left!;
+  const rMin = width - margins.right!;
```

Comment:
why this change?

Reply:
The `getDomainMargins` function in VerticalBarChart already includes these values in the `margins`. The updated margins are then passed to the axis utilities from CartesianChart, removing the need for a direct dependency on the `barwidth` prop.